src/software/backend/replay_logging/replay_logger.cpp

```diff
@@ -0,0 +1,85 @@
+#include "software/backend/replay_logging/replay_logger.h"
+
+#include <google/protobuf/util/delimited_message_util.h>
+
+#include <fstream>
+
+#include "software/logger/logger.h"
+
+namespace fs = std::experimental::filesystem;
+
+ReplayLogger::ReplayLogger(const std::string& out_dir_path, int _msgs_per_chunk)
+    : LastInFirstOutThreadedObserver<SensorMsg>(2000),
```

Comment:
why 2000? is it related to the number of msgs per chunk?

Reply:
this was just an arbitrarily large number to minimize the possibility that the buffer fills up and we start dropping messages. can move this to a named constant